tests/integration_tests/test_llm.py

```diff
@@ -401,21 +401,24 @@ def _verify_lm_lora_finetuning_layers(
     and Lora_B children layers for each specified projection, such that the product of Lora_A and Lora_B is a square
     matrix (with the dimensions expected_lora_in_features by expected_lora_in_features) for each specified projection.
     """
-    from peft.tuners.lora.layer import Linear
+    from peft.tuners.lora.layer import LoraLayer
 
     expected_lora_num_features_orig: tuple[int] = (expected_lora_in_features, expected_lora_out_features)
 
     file_names: list[str] = list_file_names_in_directory(directory_name=model_weights_directory)
     assert set(file_names) == set(expected_file_names)
 
     target_module_name: str
-    target_module_obj: Linear
+    target_module_obj: LoraLayer | torch.nn.Linear
```

Comment:
@arnavgarg1 Is this correct, or do we only get the `LoraLayer` type?

Reply:
@alexsherstinsky When merged, we get the `torch.nn.Linear` like in the base model, but without merging, it's actually the LoraLayer type!

Reply:
Previously, the target module layers were also torch.nn.Linear layers so we were always guaranteed that with or without merge, we'd end up with torch.nn.Linear layers.